turborepo-tests/integration/tests/dry_json/monorepo.t

```diff
@@ -182,52 +182,3 @@ Tasks that don't exist throw an error
   $ ${TURBO} run doesnotexist --dry=json
   Error: Could not find the following tasks in project: doesnotexist
   [1]
-
```

Comment:
Removing as duplicated test doesn't test any new behavior now that we run the Rust codepath in CI